src/js/index.js

```diff
@@ -1,5 +1,167 @@
-(function () {
+/*global $ */
+'use strict';
 
-  console.log('WOLOX');
+//Manejo de animaciones cuando abre el sitio
+function openBars(time){
+	$('.copyright').animate({
+		bottom: '+=200'
+	},time,function(){});
 
-})();
+	$('header').animate({
+		marginTop: '+=400'
+	},time,function(){});
+}
+
+var slide = 1;
+
+//Manejo del slideshow
+$('.backcontrols').hover(function(){
+	if ($('.copyright').css('bottom') ==='-200px'){
+		openBars(1000);
+	}
+},function(){});
+
+
+function pasar(slide){
+	$('body').css('background-image', 'url("../assets/slide' + slide + '.jpg")');
+}
+
+
+var paso;
+function slider(){
+	paso = setInterval(function(){
+		if(slide>=4) slide=0;
+		slide++;
+		pasar(slide);
+	},3000);
+}
+
+function fullscreen(){
+	$('.copyright').animate({
+		bottom: '-=200'
+	},1000,function(){});
+
+	$('header').animate({
+		marginTop: '-=400'
+	},1000,function(){});
+}
+
+function siguiente(){
+	window.clearInterval(paso);
+	$('button.play').html('▻');
+	if(slide<4){
+		slide++;
+		pasar(slide);
+	}else if(slide === 4){
+		pasar(slide=1); 
+	}
+}
+
+function anterior(){
+	window.clearInterval(paso);
+	$('button.play').html('▻');
+	if(slide>1){
+		slide--;
+		pasar(slide);
+	} else if(slide <= 1){
+		slide=4;
+		pasar(slide);
+	}
+}
+
+
+$('.play').click(function(){
```

Comment:
add spaces to the declarations inside this function, also don't add html from js, html is for structure and js for functionallity. add or remove classes only.